.github/pull_request_template.md

```diff
@@ -0,0 +1,50 @@
+# Description
+Mandatory, single line description of the pull request, written as though it was an order.
+
+Then, following a single blank line, a summary of the change and which issue is fixed, including 
+- [ ] what is the problem being solved
+- [ ] why this is the best approach
+- [ ] what are the shortcomings of this approach, if any
+It may include
+- [ ] background information, such as "Fixes #"
+- [ ] benchmark results
+- [ ] links to design documents
+- [ ] dependencies required for this change
+
+The summary may omit some of these details if this PR fixes a single ticket that includes these details. It is the reviewer's discretion. 
+
+## [Optional] Changes
+Multiple changes are not recommended, so this section should normally be omitted. Unforunately, they are sometimes unavoidable. If there are multiple logical changes, list them separately.
+
+1. `'foo'` is replaced by `'bar'`
+2. `'fizzbuzz'` is optimized for gas
+
+## Type of change
+
+Feel free to delete options that are not relevant.
+
+- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
+- [ ] Bug fix (non-breaking change which fixes an issue)
+- [ ] New feature (non-breaking change which adds functionality)
+- [ ] Chore (tech debt, improved developer experience, etc)
+- [ ] Performance improvement (either gas efficiency or computational efficiency)
+- [ ] Requires a documentation update
+
+# [Optional] How Has This Been Tested?
+
+Did you need to run manual tests to verify this change? If so, please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration
+
+# Checklist:
+
+## Code quality
```

Comment:
My 2c. It is nice when a PR is geared for the reviewer to efficiently understand code changes and make recommendations. Having this Checklist in every PR description is pretty noisy. I would vote for not including this checklist in PR descriptions.

Reply:
I like the idea of a checklist to remind developers of _what they should do_ to make it easier for the reviewer and future developers to understand the PR and its context.

I don't really care if the checklist _remains_ in the PR description. I am happy for people to hold themselves accountable for ticking the boxes in their head, and removing the checklist.

If there is an alternative means to this end, I am happy to explore it.

Reply:
I would be in favor the developers removing this checklist before posting the PR. Perhaps we can recommend that action in the template?

Reply:
Are you ok leaving it in for about a week, and putting it up to a team vote?